### ISSUE TITLE
Linting error on Windows prevents plugin usage

### ISSUE THREAD

**Issue by ruohki** [zodern/meteor-types#15]
Hey i was running into these issues:

``` 
While linting files with zodern:types for the app (Server):
   packages/types_plugin.js:434:9: Unable to find package: accounts-base@2.2.9
   at findPackagePath (packages/types_plugin.js:434:9)
   at packages/types_plugin.js:330:18
   at Array.forEach (<anonymous>)
   at loadPackages (packages/types_plugin.js:316:9)
   at Linter.processFilesForPackage (packages/types_plugin.js:88:20)

   While linting files with zodern:types for the app (Client):
   packages/types_plugin.js:434:9: Unable to find package: accounts-base@2.2.9
   at findPackagePath (packages/types_plugin.js:434:9)
   at packages/types_plugin.js:330:18
   at Array.forEach (<anonymous>)
   at loadPackages (packages/types_plugin.js:316:9)
   at Linter.processFilesForPackage (packages/types_plugin.js:88:20)

   While linting files with zodern:types for the app (Client):
   packages/types_plugin.js:434:9: Unable to find package: accounts-base@2.2.9
   at findPackagePath (packages/types_plugin.js:434:9)
   at packages/types_plugin.js:330:18
   at Array.forEach (<anonymous>)
   at loadPackages (packages/types_plugin.js:316:9)
   at Linter.processFilesForPackage (packages/types_plugin.js:88:20)
```

While doing furter investigations i noticed that under windows the package is using a weird path to locate packages:
`remoteCatalogRoot` is using the following format `/C/Users/guy/AppData/Local/.meteor`
whereas localCatalog uses normal windows path stuff.

**Comment by ruohki** [comment]
I've changed the following at line 39 of `types_plugin.js`

```
let remoteCatalogRoot = tropohouse.root;
if (process.platform == "win32") {
  remoteCatalogRoot = remoteCatalogRoot.replace(/^\/(\w)\//, "$1:\\").replace(/\//g,'\\');  
}
```

this resolves the issue for me, but a more sustainable solution would be welcome

**Comment by ruohki** [comment]
#16

**Comment by Clovel** [comment]
I also have this issue !

This issue need more visibility. Maybe a more explicit title would help. 

```logs
PS C:\Users\<user>\path\to\monorepo\apps\myApp> meteor lint
While linting files with zodern:types for the app (Server):
packages/types_plugin.js:434:9: Unable to find package: accounts-base@2.2.10
at findPackagePath (packages/types_plugin.js:434:9)
at packages/types_plugin.js:330:18
at Array.forEach (<anonymous>)
at loadPackages (packages/types_plugin.js:316:9)
at Linter.processFilesForPackage (packages/types_plugin.js:88:20)


While linting files with zodern:types for the app (Client):
packages/types_plugin.js:434:9: Unable to find package: accounts-base@2.2.10
at findPackagePath (packages/types_plugin.js:434:9)
at packages/types_plugin.js:330:18
at Array.forEach (<anonymous>)
at loadPackages (packages/types_plugin.js:316:9)
at Linter.processFilesForPackage (packages/types_plugin.js:88:20)


While linting files with zodern:types for the app (Client):
packages/types_plugin.js:434:9: Unable to find package: accounts-base@2.2.10
at findPackagePath (packages/types_plugin.js:434:9)
at packages/types_plugin.js:330:18
at Array.forEach (<anonymous>)
at loadPackages (packages/types_plugin.js:316:9)
at Linter.processFilesForPackage (packages/types_plugin.js:88:20)
```

**Comment by tosinek** [comment]
I ended up pulling the package locally to /packages and making the change that is in the pull request. Works.

**Comment by jbo023** [comment]
@tosinek  Ah yeah thats a good Idea, thanks

**Comment by zodern** [comment]
This is fixed in version 1.0.12.
